src/main/java/seedu/todo/controllers/concerns/Renderer.java

```diff
@@ -45,7 +50,7 @@ public static void renderDisambiguation(String replacedCommand, String detailedE
      * @param db
      * @param consoleMessage to be rendered in console, leave null if not needed
      */
-    public static void renderSelected(TodoListDB db, String consoleMessage, List<Task> tasks, List<Event> events) {
+    public static void renderSelectedIndex(TodoListDB db, String consoleMessage, List<Task> tasks, List<Event> events) {
```

Comment:
hmmm..... is this necessary? you can do a simple comment to the function to explain what it does.